.travis.yml

```diff
@@ -1,39 +1,46 @@
 matrix:
   include:
-    - language: java
-      sudo: false
-      before_install:
-        - "export DISPLAY=:99.0"
-        - "sh -e /etc/init.d/xvfb start"
-      script:
-        - 'mvn clean verify'
-        - 'if [[ $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" ]]; then mvn sonar:sonar -Dsonar.login=${SONAR_TOKEN} -Dsonar.projectKey="camel-lsp-server" -Dsonar.projectName="Camel LSP Server"; fi'
-      jdk: oraclejdk8
-      deploy:
-        provider: releases
-        api_key:  ${DEPLOY_TOKEN}
-        file:
-          - "target/camel-lsp-server-1.0.0-SNAPSHOT.jar"
-        skip_cleanup: true
-        overwrite: true
-        on:
-          branch:  master
-          # tags: true
-        branches:
-          only:
-            - master
-      cache:
-        directories:
-          - $HOME/.m2
-          - $HOME/.sonar/cache
-          - $HOME/.gradle/
-#      after_success:
-#	  - ./cd/before-deploy.sh
-#	  - ./cd/deploy.sh
-env: DISPLAY=:99
+  - language: java
+    sudo: false
+    before_install:
+    - export DISPLAY=:99.0
+    - sh -e /etc/init.d/xvfb start
+    script:
+    - mvn clean verify
+    - if [[ $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" ]]; then
+      mvn sonar:sonar -Dsonar.login=${SONAR_TOKEN} -Dsonar.projectKey="camel-lsp-server"
+      -Dsonar.projectName="Camel LSP Server"; fi
+    jdk: oraclejdk8
+#    deploy:
+#      provider: releases
+#      api_key: "${DEPLOY_TOKEN}"
+#      file:
+#      - target/camel-lsp-server-1.0.0-SNAPSHOT.jar
+#      skip_cleanup: true
+#      overwrite: true
+#      on:
+#        branch: master
+#      branches:
+#        only:
+#        - master
+    cache:
+      directories:
+      - "$HOME/.m2"
+      - "$HOME/.sonar/cache"
+      - "$HOME/.gradle/"
+    after_success:
+      - ./cd/before-deploy.sh
+      - ./cd/deploy.sh
+env:
+  matrix:
```

Comment:
why a "matrix" has been introduced?

Reply:
tbh I have no clue why :D feel free to improve it.